### PR TITLE
search: mark more strings for translation

### PR DESCRIPTION
**Pull request by xrmx**

### CODE REVIEW

**Comment by humitos** [comment]
@safwanrahman can you take a look at this file? I know that you were working on the search. These changes don't look too complicated but maybe this file won't exist anymore or similar. You know that :)

File: readthedocs/templates/search/elastic_search.html [hunk 1]
```diff
@@ -79,7 +79,7 @@ <h5>Version</h5>
 
     {% if facets.taxonomy %}
 
-      <h5>Taxonomy</h5> 
+      <h5>{% trans 'Taxonomy' %}</h5>
```
Comment (safwanrahman) [inline readthedocs/templates/search/elastic_search.html]: This is not needed. as we never did Taxonomy and it will be removed

**Comment by safwanrahman** [comment]
thanks @humitos for mentioning me.
This file will exist in the search upgrade, but I am hesitate to merge this before we deploy our new upgraded search. The template is changed in the new search, so I think we can hold this PR untill our `search_upgrade` branch merged into master.

**Comment by xrmx** [comment]
Please either merge this now, cherry-pick in your branch, close the pr but don't expect me to respin  sometime in the future when some branch will merge, thanks.

**Comment by safwanrahman** [comment]
@xrmx I have added this PR to the search Improvement milestone, so it will not get lost. If you want to merge it now, then you can open the PR against `search_upgrade` branch.

**Comment by ericholscher** [comment]
Gonna merge this, as it's useful, and the search upgrade is blocked currently :(

**Comment by ericholscher** [comment]
Thanks @xrmx

**Pull request merged**